base/reduce.jl

```diff
@@ -396,7 +396,11 @@ end
 
 
 ## prod
+"""
+    prod(f, iter)
```

Comment:
That's the weird part though, I _don't_ have `::Callable` in the docstring I added.